experiments/simple_example_mnist_loss/measure_loss.sh

```diff
@@ -0,0 +1,37 @@
+# This script will run an MNIST training workload and save measurements as CSV and metadata as JSON.
```

Comment:
this is a nice script, but I'm not entirely sure if we need to include it in the repo, given that your README and the self-documenting FLAGS and code seem to provide all the information in here (plus, users will likely have their own ways of stringing together their experiment running framework with our codebase).

Reply:
Agreed! I removed the `measure_loss.sh` script, and I removed the `plot_loss.py` script for the same reason.